cli/js/lib.deno.ns.d.ts

```diff
@@ -71,11 +71,37 @@ declare namespace Deno {
   /** Exit the Deno process with optional exit code. */
   export function exit(code?: number): never;
 
-  /** Get process priority */
+  /** **UNSTABLE**: Might not use all-caps. */
+  export enum OsPriority {
+    LOW = 19,
+    BELOW_NORMAL = 10,
+    NORMAL = 0,
+    ABOVE_NORMAL = -7,
+    HIGH = -14,
+    HIGHEST = -20
+  }
+
+  /**
+   * Returns the scheduling priority for the process specified by pid.
+   * If pid is not provided or is 0, the priority of the current process is returned.
+   */
   export function getPriority(pid?: number): number;
 
-  /** Set process priority */
-  export function setPriority(priority: number, pid?: number): void;
+  /**
+   * Attempts to set the scheduling priority for the process specified by pid.
+   * If pid is not provided or is 0, the process ID of the current process is used.
+   * The priority input must be an integer between -20 (high priority) and 19 (low priority).
+   * Due to differences between Unix priority levels and Windows priority classes,
+   * priority is mapped to one of six priority constants in os.constants.priority.
```

Comment:
Need to change `os.constants.priority` to the Deno enum

Reply:
thx, fixed